gravitee-apim-console-webui/src/portal/components/tree-component/tree.component.ts

```diff
@@ -28,6 +28,14 @@ export interface SectionNode {
   children?: SectionNode[];
 }
 
+type ActionType = 'create' | 'edit';
+
+export interface OpenMenuEvent {
+  node: SectionNode;
+  action: ActionType;
+  itemType: PortalNavigationItemType;
+}
+
```

Comment:
WDYT of this so that it matches the emitter a bit closer?
```suggestion
type NodeMenuActionType = 'create' | 'edit';

export interface NodeMenuActionEvent {
  node: SectionNode;
  action: NodeMenuActionType;
  itemType: PortalNavigationItemType;
}

```